README.adoc

```diff
@@ -8,25 +8,19 @@
 image:{img-build-status}[Build Status, link={uri-build-status}]
 image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 
-*This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-gcovr-gh-actions-sc[SonarCloud]!*
+* The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-gcovr-gh-actions-sc[SonarCloud]!*
 
 :note-caption: :information_source:
-NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
-
-It is very easy to analyze a C, C++ and Objective-C project and compute its *test coverage* with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
-
-. Create a `sonar-project.properties` file to store your configuration
-. In your `.github/workflows/build.yml` file:
-.. Make sure you have gcovr v5.0 (v5.1 is not supported, see https://jira.sonarsource.com/browse/SONAR-16269[SONAR-16269])
-.. Download the https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/[Sonar Scanner]
-.. Download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
-* add `--coverage` compilation flag to instrument the binaries
-.. Run the test binaries to count covered lines
-.. Run `gcovr --sonarqube` to produce the SonarCloud-compatible XML coverage report
-.. Run `sonar-scanner` as the final step
-* Point the `sonar.coverageReportPaths` property to the generated coverage-report file
-. Make sure that you have your token stored as a secret in your repository (`SONAR_TOKEN`). You can request new tokens using https://sonarcloud.io/account/security/[Account/Security page].
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?language=&page=1&q=topic%3Aconfiguration&sort=&type=all[configuration examples] are available here. +
```

Comment:
I am not sure I understood the purpose of the "configuration" topic, IMO it should include the coverage examples as well. To point users to the examples of analysis configuration you can refer to the bottom of this readme file.